SWIPs/swip-bzz-handshake.md

```diff
@@ -0,0 +1,69 @@
+---
+SWIP: <to be assigned>
+title: Swarm node implementer spec - 
+author: Louis Holbrook @nolash <dev@holbrook.no>
+status: Draft
+type: Track Specs
+created: 2019-08-08
+---
+
+## Abstract
+
+This SWIP is a part of a general specification of a Swarm node. The SWIP system is used for convenience only.
+
+## Motivation
+
+Enable other implementations of the swarm node.
+
+## Backwards Compatibility
+
+N/A
+
+## Test Cases
+
+N/A
+
+## Implementation
+
+N/A
```

Comment:
remove `N/A` sections

Reply:
The SWIP rules say that these fields are compulsory, so I think I need to let them stay.